RS: Auto Tiering/Redis Flex doc updates (#2238)
redis/docs: content/operate/rs/databases/flash/_index.md

  - [Probabilistic data structures]({{< relref "/operate/oss_and_stack/stack-with-enterprise/bloom" >}})

> {{<note>}} Enabling Redis Flex for [Active-Active distributed databases]({{<relref "/operate/rs/databases/active-active">}}) requires validating and getting the Redis technical team's approval first . {{</note>}}

Reviewer: Maybe make this a bit more AV + 2nd person?
```suggestion
{{<note>}}You should get the Redis technical team's approval before you enable Redis Flex for [Active-Active distributed databases]({{<relref "/operate/rs/databases/active-active">}}). {{</note>}}
```
